build.gradle.kts

```
@@ -69,6 +69,50 @@ tasks.register("publishMiraiCoreArtifactsToMavenLocal") {
     )
 }
 
+tasks.register("verifyCompiledClasses") {
+    val projects = listOf("mirai-core-api", "mirai-core-utils", "mirai-console", "mirai-console-terminal")
+    group = "verification"
+    doFirst {
+        projects.asSequence().mapNotNull {
+            runCatching { project(it) }.getOrNull()
+        }.map { project ->
+            val classesDir = project.buildDir.resolve("classes")
+            listOf(
+                listOf("kotlin/main", "kotlin/jvm/main") to listOf("runtimeClasspath", "jvmRuntimeClasspath"),
+                listOf("kotlin/main", "kotlin/android/main") to listOf(
+                    "runtimeClasspath",
+                    "androidRuntimeClasspath"
+                )
+            ).map { (targetClasses, classpath) ->
```

```suggestion
                ).map { (targetClasses: List<String>, classpath: List<String>) ->
```

这个代码对可读性是灾难，如果一定要这么做，请最起码加上类型声明，不过，还是十分希望把这部分代码拿出来成为单独的方法。另外，`runtimeClasspath`是`Configuration`的名字，建议叫`configurationNames`。